src/components/TheMainLayout.tsx

```diff
@@ -1,6 +1,6 @@
-"use client";
+"use client"
```

Comment:
Look like our code format is different so we should config `prettierrc` for this project

Reply:
Ahaa we should